packages/framework/src/Model/Category/CategoryEvent.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Category;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+class CategoryEvent extends Event
+{
+    /**
+     * The CREATE event occurs once a category was created.
+     *
+     * This event allows you to run jobs dependent on the category creation.
+     */
+    public const CREATE = 'category.create';
```

Comment:
We have to probably split the event and the event names into separate classes (class and enum) because the event now has a constructor and may return the value of the dispatched category (flag, brand, etc.). 

I'm not saying it's wrong, but I would like to evaluate the solution in a separate task 😇